core/templates/pages/exploration-editor-page/feedback-tab/services/thread-data-backend-api.service.spec.ts

```diff
@@ -111,7 +111,7 @@ describe('retrieving threads service', () => {
         suggestion_type: 'edit_exploration_state_content',
         target_id: 'exp1',
         target_type: 'exploration',
-      }
+      } as unknown as SuggestionBackendDict
```

Comment:
This seems odd. Why two casts here?

Reply:
I have done this to avoid strict type checks to some extent, similar to https://github.com/oppia/oppia/blob/23047b59eea0e29fb46f9cd651398c75d4b15517/core/templates/pages/exploration-editor-page/modal-templates/save-version-mismatch-modal.component.spec.ts#L91